src/stores/UserMediaStore.js

```diff
@@ -19,7 +22,22 @@ export default {
         },
         setMediaDeviceError(state, error) {
             Vue.set(state.mediaDeviceErrors, error.type, error.message)
-        }
+        },
+        clearMediaDeviceError(state) {
+          state.mediaDeviceErrors = {};
+        },
+        toggleAudio(state) {
```

Comment:
Rename to toggleAudioActive because toggleAudio can mean more than only on or off